tools/lms-question-converters/README.md

```diff
@@ -0,0 +1,8 @@
+## Convert Learning Management System Questions to PrairieLearn
```

Comment:
Maybe just `question_converters` as a directory name? We've been a bit inconsistent with snake_case versus kebab-case, but `node_modules` is probably a good reference.

Reply:
Done.